source/_components/binary_sensor.command_line.markdown

```diff
@@ -23,12 +23,12 @@ To use your Command binary sensor in your installation, add the following to you
 # Example configuration.yaml entry
 binary_sensor:
   - platform: command_line
-    command: cat /proc/sys/net/ipv4/ip_forward
+    command: 'cat /proc/sys/net/ipv4/ip_forward'
 ```
 
```

Comment:
Why do not you put the warning / note here? With this: <p class='note'> </p>?

Like:
```suggestion
<p class='note'>
It's highly recommended to enclose the command in single quotes `'` as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: `''`.
</p>
```
I wonder if you should put the (above) text in the description of the configuration, I think this is not really the intention.

Reply:
You mean move the warning / note from the configuration variable description to the basic example?

Reply:
Change it to my suggestion, the warning is not something to put in the description of a variable.

Reply:
OK